ssdutil/main.py

```diff
@@ -6,21 +6,61 @@
 #
 
 try:
-    import argparse
     import os
     import sys
+    import argparse
+    import psutil
+    from blkinfo import BlkDiskInfo
```

Comment:
Is "blkinfo" already installed? when I test the change with the latest 202405 branch, I see the following error:

```
admin@sonic:/usr/local/lib/python3.11/dist-packages/ssdutil# ssdutil 
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/ssdutil/main.py", line 13, in <module>
    from blkinfo import BlkDiskInfo
ModuleNotFoundError: No module named 'blkinfo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/ssdutil", line 5, in <module>
    from ssdutil.main import ssdutil
  File "/usr/local/lib/python3.11/dist-packages/ssdutil/main.py", line 17, in <module>
    raise ImportError("%s - required module not found" % str(e))
ImportError: No module named 'blkinfo' - required module not found
```

Reply:
blkinfo was added to host as part of this PR: sonic-buildimage#19362 and is available in latest master and 202405+ images.

Reply:
tested with the latest master, it works.